.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+    "semi": true,
+    "singleQuote": true,
+    "jsxSingleQuote": false,
+    "printWidth": 100,
+    "tabWidth": 2,
+    "useTabs": false,
+    "arrowParens": "avoid",
+    "endOfLine": "lf",
+    "jsxBracketSameLine": false,
+    "trailingComma": "es5",
+    "bracketSpacing": false,
+    "proseWrap": "always",
+    "quoteProps": "as-needed"
+}
```

Comment:
Add empty line in the end of every file. It is useful for git system and required as a best practise

Reply:
Prettier eats my empty line when I go to save its. How can I solve this problem?